sidecar_acceptor.go

```diff
@@ -777,6 +777,15 @@ func (a *SidecarAcceptor) getSidecarAsOrder(o order.Nonce) (order.Order, error)
 func (a *SidecarAcceptor) sendRejectBatch(batchID []byte, batch *order.Batch,
 	failure error) error {
 
+	if batch != nil {
+		// As we're rejecting this batch, we'll now cancel all funding shims
+		// that we may have registered since we may be matched with a distinct
+		// set of channels if this batch is repeated.
+		if err := a.removeShims(batch); err != nil {
```

Comment:
:+1:

Reply:
Ah yeah makes sense, otherwise we'll only cancel the shims the _next_ time a batch is proposed.